srcpkgs/mit-scheme-c/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'mit-scheme-c'
+pkgname=mit-scheme-c
+version=9.2
+revision=1
+build_wrksrc="src"
+short_desc="Implementation of the Scheme programming language"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://www.gnu.org/software/mit-scheme/"
+distfiles="${GNU_SITE}/mit-scheme/stable.pkg/${version}/mit-scheme-c-${version}.tar.gz"
+checksum=4f6a16f9c7d4b4b7bb3aa53ef523cad39b54ae1eaa3ab3205930b6a87759b170
+nocross=yes
```

Comment:
put the error message or a brief explaination

Reply:
````
$ ./xbps-src -a aarch64 pkg mit-scheme-c
=> mit-scheme-c-9.2_1: running do_build ...
run_cmd: ./configure --prefix=/builddir/mit-scheme-c-9.2/src/boot-root --enable-native-code=c --disable-host-scheme-test
checking whether make sets $(MAKE)... yes
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for gcc... aarch64-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... configure: error: in `/builddir/mit-scheme-c-9.2/src':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
See `config.log' for more details
=> ERROR: mit-scheme-c-9.2_1: do_build: 'etc/make-liarc.sh --prefix=/usr' exited with 1
=> ERROR:   in do_build() at srcpkgs/mit-scheme-c/template:15
````

Reply:
I was having trouble getting it to build with the host compiler and then compile itself using that built compiler.

Reply:
@abenson take a decision.